internal/model/recommendation_set.go

```diff
@@ -40,7 +40,7 @@ func (r *RecommendationSet) GetRecommendationSets(orgID string, orderQuery strin
 	query := db.Table("recommendation_sets").Joins(`
 		JOIN (
 			SELECT workload_id, MAX(monitoring_end_time) AS latest_monitoring_end_time 
-			FROM recommendation_sets GROUP BY workload_id
+			FROM recommendation_sets GROUP BY workload_id, container_name
```

Comment:
Without this change, we were getting only one record by latest_monitoring_end_time,
this now gives us record by latest_monitoring_end_time per container